javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,42 @@
+//A future age calculator
+const yearOfBirth = 1986
+const yearFuture = 2027
+my = yearFuture - yearOfBirth
+console.log("You will be " + my + " in 2027")
+
+//A dog age calculator 
+const dogAge = 1957
+const humanAge = 2017
+const futureAge = 2027
+const resultInDogYear = true
+const resultInHumanYear = false
```

Comment:
You should have these three variables `dogYearFuture` `dogYear` and `shouldShowResultInDogYears`
where dogYear would be equal to `(dogYearFuture - dogYearOfBirth) * 7`